hub.util.exceptions.MetaAlreadyExistsError
==========================================

.. currentmodule:: hub.util.exceptions

.. autoexception:: MetaAlreadyExistsError